docs: contact classes (#36)
starkware-libs/starknet-docs: docs/State/starknet-state.md

- $\text{storage\_root}$ is the root of another Merkle-Patricia tree of height 251 that is constructed from the contract’s storage
- $h$ is the [Pedersen](../Hashing/hash-functions#pedersen-hash) hash function.

:::info

> Note that, as of the time of writing, the commitment does not include classes information. This will be added in future versions.

Reviewer: ```suggestion
Currently, the commitment does not include information for classes. This information will be added in future versions.
```
